### PR TITLE
upgrade versions of rhai & tokio

### PR DESCRIPTION
**Pull request by PsiACE**
- rhai 0.19.0+
- tokio 0.3.5+

### CODE REVIEW

**Comment by PsiACE** [comment]
error[E0706]: functions in traits cannot be declared `async`

Oddly enough, it seems possible to pass CI under Linux, and in one of my repository commits, I once passed all CI.

File: Cargo.toml [hunk 1]
```diff
@@ -13,18 +13,18 @@ keywords = ["auth", "authorization", "rbac", "acl", "abac"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-regex = "1.3.9"
-rhai = { version = "0.19.0", features = ["sync", "only_i32", "no_function", "no_float", "no_optimize", "no_module", "no_closure", "serde", "unchecked"] }
+regex = "1.4.0"
+rhai = { version = "0.19.0", features = ["sync", "only_i32", "no_function", "no_float", "no_optimize", "serde", "unchecked"] }
```
Comment (GopherJ) [inline Cargo.toml:17]: why do we remove `no_module` and `no_closure`? I thought it's not useful for our use case

Comment (PsiACE) [reply Cargo.toml:17]: `no_function` also means `no_closure`. In addition, I cannot compile rhai 0.19.6 locally if I enable both `no_function` and `no_module`.

Comment (GopherJ) [reply Cargo.toml:17]: @PsiACE yes rhai has some upstream issues

Comment (PsiACE) [reply Cargo.toml:17]: we can simply wait for rhai 0.19.7

Comment (schungx) [reply Cargo.toml:17]: Alright, I'll put out `0.19.7` within today or tomorrow...  I've included a CI test which will make sure it compiles for your combination of features...

Comment (schungx) [reply Cargo.toml:17]: `0.19.7` released to `crates.io`.

Comment (GopherJ) [reply Cargo.toml:17]: @schungx thanks! @PsiACE could you upgrade rhai?

Comment (PsiACE) [reply Cargo.toml]: @GopherJ Done.

**Comment by GopherJ** [comment]
thanks

**Pull request merged**